spec/core/matchers/matchersUtilSpec.js

```diff
@@ -524,6 +524,20 @@ describe("matchersUtil", function() {
       expect(jasmineUnderTest.matchersUtil.equals(mapA, mapB)).toBe(false);
     });
 
+    it("passes for ArrayBuffers with same length and content", function() {
```

Comment:
These specs should do something similar to the existing Typed Array specs in pretty printer so they don't break in an environment where `ArrayBuffer` isn't defined.